frontend/src/views/fixed-nav-sticky-footer-view.js

```diff
@@ -0,0 +1,87 @@
+import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import '../components/ex-card.js';
+
+/**
+ * `fixed-nav-sticky-footer-view`
+ *
+ * FixedNavStickyFooterView element.
+ *
+ * @customElement
+ * @polymer
+ */
+class FixedNavStickyFooterView extends PolymerElement {
+
+    static get template() {
+        return html`
+            <style include="common-styles">
+                :host {
+                    display: flex;
+                    flex-direction: column;
+                    height: 100%;
+                }
+                
+                .wrapper {
+                    margin: 0 auto; /* Horizontally center the layout grid if there is space around it */
+                    max-width: 960px;
+                    padding: 0 var(--lumo-space-m);
+                }
+                
+                #header {
+                    z-index: 1;
+                    position: fixed;
+                    right: 0;
+                    left: 0;
+                    display: flex;
+                    white-space: nowrap;
+                    flex-wrap: wrap;
+                    align-items: center;
+                    padding: 0 var(--lumo-space-s);
+                    --lumo-primary-text-color: var(--lumo-contrast-90pct);
+                }
+                
+                .searchField {
+                    padding: var(--lumo-space-s) 0;
+                    margin-right: var(--lumo-space-s);
+                }
+                
+                #header a,
+                #header span {
```

Comment:
Not sure if any `span` it used, at least I was unable to find it. Consider removing this line.

Reply:
Good point. There used to be a span (like the "Disabled" item here: https://getbootstrap.com/docs/4.3/examples/sticky-footer-navbar/) but I removed it since it didn't make much sense to me. Will remove this unnecessary selector.

Reply:
Done.